components/treeview/metadata/treeview.yml

```diff
@@ -334,9 +334,9 @@ examples:
         </li>
       </ul>
 
-  - id: treeview
+  - id: treeview-thumbnail-quiet
```

Comment:
Do the `.spectrum-Thumbnail-image-wrapper` items here need to have `.spectrum-OpacityCheckerboard` added to them, as well?

Reply:
Thanks. The images were working locally but it seems they needed the starting slash removed to work on the PR site. Both of these are now updated.